clientdb/order_test.go

```diff
@@ -43,6 +43,12 @@ func TestSubmitOrder(t *testing.T) {
 	}
 	o.Details().MinUnitsMatch = 10
 	o.Details().ChannelType = order.ChannelTypeScriptEnforced
+
+	// It is not possible for a oder to have AllowedNodeIDs and
+	// NotAllowedNodeIDs at the same time but we want to test
+	// serialization/deserialization here.
+	o.Details().AllowedNodeIDs = [][33]byte{{1, 2, 3}}
```

Comment:
we should add at least two IDs to make sure multiple entries can be stored and retrieved correctly.